packages/core/src/model.js

```diff
@@ -1749,6 +1749,10 @@ ${associationOwner._getAssociationDebugList()}`);
    */
   static async create(values, options) {
     options = cloneDeep(options) ?? {};
+    const association = this._searchInclude(values);
+    if (association.length > 0 && !options.include) {
+      options.include = association;
```

Comment:
only call `_searchInclude` if `options.include` is not set, to avoid computing it if not necessary

The `association.length > 0` check should not be necessary and can be removed

`association` should be plural, but entire code can be simplified to this:

```
options.include ??= getDefaultCreateInclude(this, values);
```